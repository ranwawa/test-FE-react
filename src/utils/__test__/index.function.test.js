import {isMobile, isPwd, REG_MOBILE, REG_PWD } from '..';

describe('isMobile', () => {
	test('13333333333应该返回true', () => {
		expect(isMobile('13333333333')).toBe(true);
	});

	test('1333333应该返回false', () => {
		expect(isMobile('1333333')).toBe(false);
	})
});


describe('isPwd', () => {
	test('123456应该返回true', () => {
		expect(isPwd(123456)).toBeTruthy();
	});

	test('123应该返回false', () => {
		expect(isPwd('123')).toBeFalsy();
	})
});

describe('snapshot', () => {
	test('手机号码的正则表达式', () => {
		expect(REG_MOBILE).toMatchSnapshot();
	})

	test('密码的正则表达式', () => {
		expect(REG_PWD).toMatchSnapshot();
	})
})
