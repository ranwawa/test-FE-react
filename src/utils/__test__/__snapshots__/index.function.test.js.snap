// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 密码的正则表达式 1`] = `/\\.\\{6\\}/`;

exports[`snapshot 手机号码的正则表达式 1`] = `/1\\[3-8\\]\\\\d\\{9,9\\}/`;
